Automattic/vip-cli
# FORNO-555: Add start media import command

## Description

Adds a new `vip media-import` command to start a new media import

## Steps to Test

Make sure the docker containers for GOOP, Parker and MMS are running locally on your computer.

1. Check out PR.
1. Run `npm run build`
1. Run `VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip-import-media.js @1000.production --url https://beruberuberu-test.go-vip.co/wp-content/uploads/2021/03/wp-content.zip`
1. Verify the command is showing showing the correct environment and URL
1. Check MMS logs to confirm that files are being uploaded via the mock file service
1. Verify the command stops checking status after the import is `COMPLETED`
